tests/test_runtime.py

```
@@ -26,3 +30,323 @@ def test_recognizes_known_player_death(self) -> None:
+            runtime._stop.set()
+            raise Exception("stopped")
+
+        fake_docker = MagicMock()
+        fake_docker.from_env.side_effect = stop_after_first
+
+        with patch("minecraft_manager.runtime.threading.Timer"):
+            with patch.dict("sys.modules", {"docker": fake_docker}):
+                try:
+                    runtime._logs()
+                except Exception:
+                    pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)